src/scripts/main.js

```diff
@@ -2,7 +2,29 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+for (let i = 0; i < people.length; i++) {
+  table.insertRow().insertAdjacentHTML('afterbegin', `
+    <tr>
+        <td>
+            ${people[i].name}
+        </td>
+        <td>
+            ${people[i].sex === 'm' ? 'Male' : 'Female'}
+        </td>
+        <td>
+            ${people[i].born}
+        </td>
+        <td>
+            ${people[i].died}
+        </td>
+        <td>
+            ${people[i].died - people[i].born}
+        </td>
+        <td>
+            ${Math.ceil(people[i].died / 100)}
+        </td>
```

Comment:
why do you have 4 indents inside template string?

Reply:
Fixed indents